envsetup.sh

```diff
@@ -214,11 +214,17 @@ fi # if -e ${OE_ENV_FILE}
 # UPDATE_ALL() - Make sure everything is up to date
 ###############################################################################
 yoe_update_all() {
-  git submodule update
+  CWD=`pwd`
+  cd ${OE_BASE}
+  git pull && git submodule sync && git submodule update
+  cd $CWD
```

Comment:
what do you think of the following pattern:

cd ${OE_BASE}
do something
cd -

This is a patten I use, but may not be reliable if the "do something" has a cd in it.